pkgs/development/embedded/fpga/trellis/default.nix

```diff
@@ -1,30 +1,27 @@
-{ lib, stdenv, fetchFromGitHub
-, python3, boost
-, cmake
-}:
-
-stdenv.mkDerivation rec {
-  pname = "trellis";
-  version = "2021.07.06";
+{ lib, stdenv, fetchFromGitHub, python3, boost, cmake }:
 
+let
+  rev = "03e0070f263fbe31c247de61d259544722786210";
   # git describe --tags
-  realVersion = with lib; with builtins;
-    "1.0-482-g${substring 0 7 (elemAt srcs 0).rev}";
+  realVersion = "1.0-532-g${builtins.substring 0 7 rev}";
+in stdenv.mkDerivation rec {
+  pname = "trellis";
+  version = "2021-09-01";
```

Comment:
Don't change the version number format. Just keep the dots.

Reply:
The standard in Nixpkgs seems to be YYYY-MM-DD, i.e. ISO 8601.

Reply:
In general I'm hesitant to change version numbers even if they're inconsistent, because sometimes it interferes with how `nix-env` works when doing upgrades, but admittedly that's probably no worse of an "API change" than changing the arglist is, as an example.

The main reason it's like this is because in the past Nixpkgs definitively did not have any kind of standards for this thing, some people might have just used ISO 8601 coincidentally, and the zerover-style convention was one I had seen before.

I'd like to change this in the other expressions too I guess, but really the whole EDA subsection of Nixpkgs needs a little bit of a reorganization anyway. So I'll accept this and won't block this change on it.